gene2phenotype_project/gene2phenotype_app/views/user.py

```diff
@@ -97,11 +97,17 @@ class CreateUserView(generics.CreateAPIView):
 
         Usage:
             Send a POST request with the required user details (username, email, 
-            password, first_name, last_name) to this API to create a new user account.
+            password, first_name, last_name, panels) to this API to create a new user account.
     """
 
     serializer_class = CreateUserSerializer
-    permission_classes = [permissions.IsAuthenticated]
+    permission_classes = [permissions.IsAdminUser]
+
+class AddUserToPanelView(generics.CreateAPIView):
+    serializer_class = AddUserToPanelSerializer
+    permission_classes = [permissions.IsAdminUser]
```

Comment:
`IsAdminUser` denies permission to any user unless `is_staff` is True.
In other endpoints we are using the `is_superuser` flag (IsSuperUser from base). 
In the end both are valid, but if we use a different permission for user_panel then we should leave this documented on confluence and also add a comment here.

Reply:
Yeah i thought we agreed that for creating user and adding user to a panel to admin (us) only, I can change it but just wanted to make sure the details are correct because right now, I have implemented that `only is_staff=True (AdminUser)` can create user or add user to a panel

Reply:
Yeah it's ok, we can just add the details to [this page](https://www.ebi.ac.uk/seqdb/confluence/display/EBIMedical/Browsable+API%3A+Endpoint+Documentation#BrowsableAPI:EndpointDocumentation-Endpointstomanagelogin).